client/me/help/help-contact/index.jsx

```diff
@@ -202,6 +207,8 @@ const HelpContact = React.createClass( {
 			email
 		);
 
+		analytics.tracks.recordEvent( 'calypso_help_contact_directly_ask_question' );
```

Comment:
This could either go in the data layer middleware or keep it here but we should dispatch the analytics `recordEvent` action instead.